src/dart/parse.rs

```diff
@@ -154,14 +168,19 @@ impl<'a> Parser<'a> {
         Ok(())
     }
 
+    /// Returns true and consumes the current token
+    /// if that matches the given punctuation pair.
     pub fn eat_punctuation2(&mut self, c1: char, c2: char) -> bool {
         self.try(|p| p.expect_punctuation2(c1, c2)).is_some()
     }
 
+    /// Returns true if the current token is the given punctuation pair.
```

Comment:
Inconsistent use of "matches" and "is". Stick to one or the other. Hint: the former.